### PR TITLE
Revendor MAO

### PR DESCRIPTION
**Pull request by alexander-demichev**
Revendor MAO for dropping the requirement to set cluster ID label manually.

openshift/machine-api-operator#608

### CODE REVIEW

**Comment by enxebre** [comment]
https://github.com/openshift/cluster-api-provider-gcp/pull/98#discussion_r441454340

File: go.mod298498081.tmp [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module sigs.k8s.io/cluster-api-provider-azure
```
Comment (JoelSpeed) [inline go.mod298498081.tmp:1]: Not sure this should be checked in? Any idea where this file came from?

Comment (alexander-demichev) [reply go.mod298498081.tmp:1]: That's vscode

Comment (alexander-demichev) [reply go.mod298498081.tmp]: I removed it

**Pull request merged**